core/tests/test_utils.py

```diff
@@ -3555,7 +3593,7 @@ def create_story_for_translation_opportunity(
             owner_id, topic_id, story.id)
         topic_services.publish_story(
             topic_id, story.id, admin_id)
-        story_services.update_story(
+        topic_services.update_story_and_topic_summary(
```

Comment:
I think you can still call this update_story (otherwise it's not clear with the new name whether we're updating the topic summary and story summary, or the topic summary + story). In any case, I think updating the summaries should be considered a consequence of updating the main object, and not a separate independently-doable action.

Reply:
Done.